malariagen_data/anoph/gplt_params.py

```diff
@@ -36,6 +36,13 @@
     "Plot height in pixels (px).",
 ]
 
+row_height: TypeAlias = Annotated[
+    int,
+    "Plot height per row (sample) in pixels (px).",
+]
+
+row_height_default: row_height = 7
```

Comment:
This feels quite specific to the CNV heatmap plotting function, which is the only place where it's used I think. I reckon better not to define here, rather can just define within the function signature.

Reply:
OK, will do. 
`7` was the default `row_height` for both `plot_cnv_hmm_heatmap` and `plot_cnv_hmm_heatmap_track`.

`plot_frequencies_heatmap` has a default `row_height` of `20`.

`plot_aim_heatmap` has a default `row_height` of `4`.